### ISSUE TITLE
Fixed Product Tax element bugs on product edit form

### ISSUE THREAD

**Issue by kate-kyzyma** [magento/magento2#20017]
### Preconditions (*)

1. Magento 2.3-develop

### Steps to reproduce (*)

1. Create Fixed Product Tax attribute 
2. Assign the newly created attribute to attribute set
3. Open product edit form

### Expected result (*)

The Fixed Product Tax field works properly. The FPT attribute label has no asterisk, considering the FPT cannot be required field. Basically, the flow should be the same as on Magento 2.3.
![screen recording 2018-12-30 at 11 33 46 am](https://user-images.githubusercontent.com/32304009/50546341-12357400-0c2f-11e9-9897-096cb94b3a39.gif)

### Actual result (*)

1. (Reported initially but does not occur at present: The FPT attribute field is marked as required. However, this is not the required attribute.)
2. The attribute label disappears once at least one row is added.
3. (Reported initially but does not occur at present: Tax and Website fields are misaligned within the row.)
![screen recording 2018-12-30 at 11 32 37 am](https://user-images.githubusercontent.com/32304009/50546347-1bbedc00-0c2f-11e9-98e0-529aa418ed9b.gif)

**Comment by magento-engcom-team** [comment]
Hi @kate-kyzyma. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@kate-kyzyma do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [x] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [x] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [x] 6. Add label `Issue: Confirmed` once verification is complete. 

- [x] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by dmytro-ch** [comment]
@magento-engcom-team give me 2.2-develop instance

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch. Thank you for your request. I'm working on Magento 2.2-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, here is your [Magento instance](https://i-20017-2-2-develop.instances.magento-community.engineering).
Admin access: [https://i-20017-2-2-develop.instances.magento-community.engineering/admin](https://i-20017-2-2-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by magento-engcom-team** [comment]
@dmytro-ch Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-97383` were created

**Comment by dmytro-ch** [comment]
The issue can be reproduced in **2.2-develop** branch.

<img width="765" alt="screen shot 2018-12-30 at 12 39 33 pm" src="https://user-images.githubusercontent.com/11693779/50546473-e1563e80-0c30-11e9-9b37-0ef515688e8d.png">
<img width="738" alt="screen shot 2018-12-30 at 12 45 43 pm" src="https://user-images.githubusercontent.com/11693779/50546474-e1563e80-0c30-11e9-82d8-d5f0a5651a06.png">


The issue is not reproducible for **Magento 2.3**.

Thank you.

**Comment by irajneeshgupta** [comment]
@magento-engcom-team give me 2.2-develop instance

**Comment by magento-engcom-team** [comment]
Hi @irajneeshgupta. Thank you for your request. I'm working on Magento 2.2-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @irajneeshgupta, here is your [Magento instance](https://i-20017-2-2-develop.instances.magento-community.engineering).
Admin access: [https://i-20017-2-2-develop.instances.magento-community.engineering/admin](https://i-20017-2-2-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by dmytro-ch** [comment]
Hello @irajneeshgupta,
please note the issue has been already assigned.
I will handle this issue asap.
Thank you!

**Comment by engcom-Bravo** [comment]
Fixed in 2.3-develop but only partially. (See the description. It's been updated).

**Comment by magento-engcom-team** [comment]
@kate-kyzyma, thank you for your report.     
    
Unfortunately, we are archiving this ticket now as it did not get much attention from both Magento Community and Core developers for an extended period. This is done in an effort to create a quality, community-driven backlog which will allow us to allocate the required attention more easily.

Please feel free to comment, reopen or create new ticket according to the [Issue reporting guidelines](https://github.com/magento/magento2/wiki/Issue-reporting-guidelines) 
if you are still facing this issue on the latest `2.3-develop` branch. Thank you for collaboration.
